src/test/java/org/springframework/data/redis/connection/RedisElastiCacheConfigurationUnitTests.java

```diff
@@ -23,8 +23,9 @@
  * Unit tests for {@link RedisStaticMasterReplicaConfiguration}.
  *
  * @author Mark Paluch
+ * @author Tamer Soliman
```

Comment:
@mp911de - I don't consider myself _worthy_ of adding my name as an "author" for such an utterly trivial change, but I'm only strictly following the [guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc#quickstart):
> Make sure you added yourself as author in the headers of the classes you touched.

Wouldn't be hurt if you removed it 🙂